oio/event/filters/notify.py

```diff
@@ -22,22 +23,63 @@
 
 class NotifyFilter(Filter):
 
+    @staticmethod
+    def _parse_no_replicate(array):
+        """
+        array is in this format ["urlencoded(account)",
+                                 "urlencoded(account2)/urlencoded(container2)"]
+        and we want to return this {account: [], account2: [container2]}
+        empty list means that everything is accepted
+        """
+        no_replicate = dict()
+        for a in array:
+            if '/' in a:
+                acct, cnt = a.split('/', 1)
+                acct = unquote_plus(acct)
+                cnt = unquote_plus(cnt)
+                if no_replicate.get(acct, None):
+                    no_replicate[acct].append(cnt)
+                else:
+                    no_replicate[acct] = [cnt]
+            else:
+                no_replicate[unquote_plus(a)] = []
+        return no_replicate
+
+    def _should_replicate(self, account, container):
+        if self.no_replicate is None:
+            return True
+        containers = self.no_replicate.get(account, None)
+        if containers == []:
+            return False
+        elif containers is None:
+            return True
+        elif container in containers:
+            return False
+        else:
+            return True
+
     def init(self):
         queue_url = self.conf.get('queue_url')
+        self.no_replicate = self._parse_no_replicate(
+            self.conf.get('no_replicate', []))
```

Comment:
I think "exclude" would be more explicit than "no_replicate".
Also, if `conf` is loaded from a file, there is a chance that the value is a string and not a list.